money-core/src/main/scala/com/comcast/money/core/Formatters.scala

```diff
@@ -35,4 +35,12 @@ object Formatters {
 
   def toHttpHeader(spanId: SpanId): String =
     HttpHeaderFormat.format(spanId.traceId, spanId.parentId, spanId.selfId)
+
+  def fromB3HttpHeaders(traceId: String, maybeParentSpanId: Option[String], maybeSpanId: Option[String]) = Try {
+    (maybeParentSpanId, maybeSpanId) match {
```

Comment:
unclear as to why this is in a `Try`

Reply:
make sure to define an explicit return type.

Also, I think `Try` isn't correct, we want this to return an `Option`